doc/reduce-traffic.md

```diff
@@ -6,7 +6,7 @@ Some node operators need to deal with bandwidth caps imposed by their ISPs.
 By default, Bitcoin Core allows up to 125 connections to different peers, 10 of
 which are outbound. You can therefore, have at most 115 inbound connections.
 Of the 10 outbound peers, there can be 8 full outgoing connections and 2 with
```

Comment:
suggest s/full outgoing/full-relay/ as the sentence already stated that these are outbound and the codebase (net.h/net.cpp/net_processing.cpp) and debug log call them full-relay

Reply:
Fixed both occurrences